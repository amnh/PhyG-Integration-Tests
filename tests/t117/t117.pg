--build(distance, dwag, block), inputting continuous tnt char types
set(seed:1634561640)
read("arcoida_cont.ss", "arcoida_cont2.ss")
build(distance, dwag, block, eun, displaytrees: 3)
report("t117.csv", data, overwrite)
report("t117.tre", newick, graphs, overwrite)
