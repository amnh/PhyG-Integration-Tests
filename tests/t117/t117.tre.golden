(Adacnarca_limopsoides_USNM21:16.0,(Empleconia_vaginatus_FMNH05:17.5,(Cratis_progressa:19.0,(Chama_frondosa_CR52:38.0,(Bathyarca_sp_Brasil:14.5,(Bathyarca_perversidens_AU36:19.0,(Axinactis_Glycymerella_americana_USNM22:29.0,(Hochstetteria_munita_USNM47:20.5,((Glycymeris_striatularis:26.0,(Grammatodontinae:38.0,(((Barbatia_lacerata:28.0,Glycymeris_holosericus_AU13:24.0)HTU128:6.0,(Arcopsis_solida:25.5,Glycymeris_ovatus_CL02:22.0)HTU129:2.5)HTU110:4.0,(Glycymeris_reevei:22.0,(Barbatia_tenera:29.0,(Glycymeris_scripta_FMNH17:19.5,(((Glycymeris_crebreliratus_AU40:19.0,Tucetona_flabellata:20.5)HTU154:4.0,(Tucetona_gealei_MNHN113:20.0,((Trisidos_semitorta_AU44:24.0,(Striarca_olivacea:26.0,(Trisidos_tortuosa_AU102:22.0,((Pteria_colymbus_PR37:29.5,((Philobrya_sublaevis_USNM27:21.0,(Axinactis_inaequalis:30.0,Pinna_carnea_PR24:25.0)HTU150:4.5)HTU120:8.5,(Scaphula_pinna_BM17:29.0,(Cucullaea_labiata:23.0,Senilia_senilis_FMNH20:22.0)HTU122:4.0)HTU151:9.0)HTU107:2.5)HTU121:5.0,(Limopsis_soboles_AU39:19.5,((Barbatia_foliata:25.0,((Glycymeris_formosus:32.0,(Anadara_transversa:25.0,Notomytilus_rubra:26.5)HTU160:4.0)HTU157:5.0,(Micromytilus_crenulatifera_AU140:20.0,(Anadara_floridana:31.5,Neocardia_sp_A_USNM15:17.5)HTU159:2.5)HTU158:7.5)HTU156:6.5)HTU125:5.0,(Adacnarca_nitens:21.0,Glycymeris_grayana_AU14:25.0)HTU127:6.0)HTU109:3.5)HTU126:6.5)HTU102:6.5)HTU153:6.0)HTU123:4.0)HTU152:6.5,(Stenocista_gambiensis_MNHN117:19.5,Striarca_sp_MNHN101:27.0)HTU161:5.5)HTU108:12.5)HTU155:4.0)HTU124:3.5,(Litharca_lithodomus:21.0,((Chama_sp_PR41:37.5,Lissarca_picta:22.5)HTU142:5.0,(Lissarca_rubrofusca_USNM17:22.0,((Limopsis_loringi_AU11:14.5,(Limopsis_forteradiata_AU01:14.5,Limopsis_marionensis:14.5)HTU115:3.5)HTU139:6.5,(Limopsis_woodwardi_AU26:11.0,(Limopsis_tenisoni_AU29:15.5,(Lissarca_notarcadensis_USNM14:18.5,((Lima_lima:28.0,((Bentharca_asperula_USNM68:25.5,Isognomon_radiatus_PR29:32.0)HTU134:11.0,(Lima_caribaea:29.5,(Isognomon_alatus_PR44:24.5,(Limopsis_aurita_BMNH07:12.5,(Bentharca_nodulosa:20.0,(Arcopsis_adamsi:26.0,Limopsis_diegensis:16.0)HTU138:5.5)HTU114:5.0)HTU137:11.0)HTU104:5.5)HTU135:2.5)HTU113:2.5)HTU136:10.5,(Nipponarca_bistrigata:26.0,((Cosa_parallelogramma_USNM26:20.5,Melaxinaea_labyrintha:27.5)HTU146:7.5,(Mytilid_sp:46.5,((Lithophaga_plumula:44.5,Malleus_candeanus_PR31:30.0)HTU145:9.5,(Parallelodontinae:35.5,(Noetia_ponderosa_FLA01:24.5,Philobrya_setosa_USNM41:27.5)HTU119:5.0)HTU149:8.0)HTU106:2.5)HTU147:9.0)HTU118:4.5)HTU148:5.5)HTU101:5.0)HTU141:8.5)HTU116:2.0)HTU140:2.5)HTU105:10.0)HTU143:3.5)HTU117:8.5)HTU144:11.0)HTU133:3.0)HTU131:4.5)HTU111:2.0)HTU130:3.5)HTU103:4.5)HTU132:3.0)HTU112:12.0,(Aupouria_parvula:24.5,(((Anadara_geissei_CV01:14.0,((Fugleria_illota:14.0,(Bathyarca_sp_A:16.0,(((Glycymeris_pilosa_FMNH06:19.5,(Glycymeris_glycymeris_MNHN120:18.0,(Glycymeris_lividus_cf_MNHN28:24.0,Tucetona_pectunculus:18.5)HTU163:6.0)HTU164:3.5)HTU165:9.5,(Anadara_ovalis:13.5,(Hyotissa_hyotis_CR48:29.5,((Brachidontes_domingensis_PR26:28.0,Lithophaga_aristata:14.5)HTU183:4.0,(Atrina_rigida:18.0,(Limaria_pellucida:46.5,(Isognomon_bicolor_PR33:28.0,(Pinctada_mazatlantica_CR49:18.5,Pteria_hirundo:27.0)HTU186:2.5)HTU185:3.5)HTU184:9.0)HTU187:9.0)HTU182:8.5)HTU181:14.5)HTU180:7.0)HTU166:7.0,(Scaphula_celox_:24.0,Scaphula_deltae_BMNH16:20.0)HTU169:10.0)HTU167:4.0)HTU168:10.5)HTU170:8.5,(Barbatia_domingensis:14.5,(Barbatia_amygdalumtostum:10.5,(Bentharca_xenophoricola_FMNH13:22.0,(Barbatia_candida_PR04:9.5,Barbatia_reeveana:11.5)HTU173:6.0)HTU174:7.5)HTU172:6.5)HTU179:5.5)HTU171:6.0)HTU175:9.0,(Striarca_interplicata_USNM49:18.5,(Arcopsis_afra:12.0,Sheldonella_repenta_AU33:14.0)HTU178:5.0)HTU177:6.0)HTU176:12.0,(Cosa_waikikia:21.5,(Cosa_costata:20.5,Cosa_scabra_AU155:16.0)HTU191:3.0)HTU190:6.5)HTU188:5.0)HTU189:8.0)HTU198:4.0)HTU162:6.5)HTU192:9.5)HTU193:8.0)HTU194:6.0)HTU195:7.0)HTU196:3.0)HTU197:8.0)HTU100[1217.0];
(Adacnarca_limopsoides_USNM21:16.0,(Empleconia_vaginatus_FMNH05:17.5,(Cratis_progressa:19.0,(Chama_frondosa_CR52:38.0,(Bathyarca_sp_Brasil:14.5,(Bathyarca_perversidens_AU36:19.5,(Axinactis_Glycymerella_americana_USNM22:28.0,((Hochstetteria_munita_USNM47:16.0,Philobrya_sublaevis_USNM27:15.5)HTU191:9.0,((Glycymeris_striatularis:26.5,(Grammatodontinae:38.5,((Glycymeris_ovatus_CL02:17.5,(Glycymeris_holosericus_AU13:20.5,(Barbatia_domingensis:16.0,Barbatia_lacerata:19.0)HTU157:21.0)HTU126:3.0)HTU110:3.5,(Glycymeris_reevei:19.0,(Barbatia_tenera:28.5,(Glycymeris_scripta_FMNH17:19.0,(((Glycymeris_crebreliratus_AU40:18.5,Tucetona_flabellata:21.0)HTU151:4.0,(Tucetona_gealei_MNHN113:20.5,((Trisidos_semitorta_AU44:23.0,(Striarca_olivacea:25.5,(Trisidos_tortuosa_AU102:21.5,((Pteria_colymbus_PR37:28.0,((Axinactis_inaequalis:29.0,Pinctada_mazatlantica_CR49:28.5)HTU147:7.0,(Scaphula_pinna_BM17:27.5,(Cucullaea_labiata:22.0,Senilia_senilis_FMNH20:22.0)HTU120:5.5)HTU148:7.5)HTU107:4.0)HTU119:7.5,(Limopsis_soboles_AU39:19.0,((Barbatia_foliata:26.5,((Glycymeris_formosus:33.0,Notomytilus_rubra:23.5)HTU154:6.0,(Micromytilus_crenulatifera_AU140:19.5,(Anadara_floridana:32.5,Neocardia_sp_A_USNM15:17.0)HTU156:2.5)HTU155:4.0)HTU153:7.5)HTU123:5.0,(Adacnarca_nitens:20.5,Glycymeris_grayana_AU14:25.5)HTU125:5.0)HTU109:6.0)HTU124:6.5)HTU102:6.0)HTU150:6.0)HTU121:5.0)HTU149:8.0,(Stenocista_gambiensis_MNHN117:19.5,Striarca_sp_MNHN101:27.0)HTU158:5.5)HTU108:13.0)HTU152:4.0)HTU122:3.5,(Litharca_lithodomus:21.0,((Chama_sp_PR41:37.0,Lissarca_picta:22.0)HTU139:5.0,(Lissarca_rubrofusca_USNM17:21.5,((Limopsis_loringi_AU11:14.5,(Limopsis_forteradiata_AU01:14.5,Limopsis_marionensis:14.5)HTU114:3.5)HTU136:6.5,(Limopsis_woodwardi_AU26:11.0,(Limopsis_tenisoni_AU29:15.5,(Lissarca_notarcadensis_USNM14:18.5,((Lima_lima:28.0,((Bentharca_asperula_USNM68:25.5,Isognomon_radiatus_PR29:32.0)HTU131:11.5,(Lima_caribaea:29.5,(Isognomon_alatus_PR44:24.0,(Limopsis_aurita_BMNH07:13.0,(Arcopsis_adamsi:27.0,Limopsis_diegensis:14.5)HTU135:3.5)HTU134:10.0)HTU104:5.5)HTU132:2.5)HTU113:3.0)HTU133:10.0,(Nipponarca_bistrigata:24.5,(((Lithophaga_plumula:44.5,Malleus_candeanus_PR31:29.5)HTU142:12.5,(Parallelodontinae:33.5,(Noetia_ponderosa_FLA01:23.5,(Anadara_geissei_CV01:15.5,Philobrya_setosa_USNM41:27.0)HTU146:9.5)HTU118:5.5)HTU145:5.5)HTU106:5.5,(Cosa_parallelogramma_USNM26:21.0,Melaxinaea_labyrintha:26.5)HTU143:7.5)HTU117:4.5)HTU144:6.5)HTU101:5.0)HTU138:8.5)HTU115:2.0)HTU137:2.5)HTU105:10.0)HTU140:3.5)HTU116:9.0)HTU141:11.0)HTU130:3.5)HTU128:4.5)HTU111:4.5)HTU127:4.5)HTU103:3.5)HTU129:2.5)HTU112:13.5,(Aupouria_parvula:24.0,((Anadara_transversa:20.5,(((Fugleria_illota:16.0,(Bentharca_nodulosa:12.5,(Bathyarca_sp_A:16.0,(((Glycymeris_pilosa_FMNH06:20.5,(Glycymeris_glycymeris_MNHN120:18.0,(Glycymeris_lividus_cf_MNHN28:25.0,Tucetona_pectunculus:18.5)HTU160:6.0)HTU161:3.0)HTU162:9.0,(Anadara_ovalis:13.5,(Hyotissa_hyotis_CR48:29.5,((Brachidontes_domingensis_PR26:28.5,Lithophaga_aristata:15.0)HTU180:4.0,((Limaria_pellucida:46.5,(Isognomon_bicolor_PR33:27.5,Pteria_hirundo:27.0)HTU183:3.5)HTU182:6.0,(Atrina_rigida:12.5,Pinna_carnea_PR24:13.0)HTU185:6.0)HTU184:10.5)HTU179:8.0)HTU178:15.0)HTU177:6.5)HTU163:7.0,(Scaphula_celox_:24.5,Scaphula_deltae_BMNH16:20.5)HTU166:10.5)HTU164:4.0)HTU165:8.5)HTU167:7.0)HTU168:5.0,(Barbatia_amygdalumtostum:13.0,(Bentharca_xenophoricola_FMNH13:22.5,(Barbatia_candida_PR04:10.5,Barbatia_reeveana:11.5)HTU171:5.5)HTU172:7.0)HTU170:7.5)HTU169:8.0,(Striarca_interplicata_USNM49:18.5,(Sheldonella_repenta_AU33:13.5,(Arcopsis_afra:10.5,Arcopsis_solida:10.5)HTU176:6.0)HTU175:4.0)HTU174:8.5)HTU173:4.0)HTU181:12.0,(Mytilid_sp:47.0,(Cosa_waikikia:22.5,(Cosa_costata:21.0,Cosa_scabra_AU155:16.5)HTU190:3.0)HTU189:4.5)HTU188:7.5)HTU186:5.5)HTU187:8.5)HTU198:5.5)HTU159:6.5)HTU192:7.5)HTU193:7.5)HTU194:6.0)HTU195:8.0)HTU196:3.0)HTU197:8.0)HTU100[1177.0];
(Adacnarca_limopsoides_USNM21:15.5,(Empleconia_vaginatus_FMNH05:17.0,(Cratis_progressa:18.5,(Chama_frondosa_CR52:37.5,(Bathyarca_sp_Brasil:14.0,(Bathyarca_perversidens_AU36:19.5,(Axinactis_Glycymerella_americana_USNM22:29.5,((Hochstetteria_munita_USNM47:16.0,Philobrya_sublaevis_USNM27:14.5)HTU191:6.5,((Glycymeris_striatularis:26.5,(Grammatodontinae:37.0,((Glycymeris_lividus_cf_MNHN28:22.5,(Glycymeris_ovatus_CL02:17.0,(Glycymeris_holosericus_AU13:20.0,(Barbatia_lacerata:21.0,Barbatia_reeveana:10.5)HTU159:20.5)HTU125:2.5)HTU110:3.5)HTU126:4.0,(Glycymeris_reevei:18.5,(Barbatia_tenera:29.5,(Glycymeris_scripta_FMNH17:19.0,(((Glycymeris_crebreliratus_AU40:19.0,Tucetona_flabellata:20.5)HTU152:4.0,(Tucetona_gealei_MNHN113:22.5,((Sheldonella_repenta_AU33:13.0,(Stenocista_gambiensis_MNHN117:20.0,Striarca_sp_MNHN101:26.5)HTU160:5.5)HTU149:8.0,(Trisidos_semitorta_AU44:23.5,(Striarca_olivacea:26.0,(Trisidos_tortuosa_AU102:23.0,((Axinactis_inaequalis:29.0,(Scaphula_pinna_BM17:31.0,(Cucullaea_labiata:22.5,Senilia_senilis_FMNH20:22.5)HTU120:3.0)HTU148:6.5)HTU107:4.0,(Limopsis_soboles_AU39:17.5,(Glycymeris_grayana_AU14:25.5,(Barbatia_foliata:25.0,((Glycymeris_formosus:31.5,Notomytilus_rubra:25.0)HTU156:4.0,(Micromytilus_crenulatifera_AU140:22.5,(Bathyarca_sp_A:18.0,(Anadara_floridana:28.5,Neocardia_sp_A_USNM15:21.5)HTU158:5.5)HTU155:6.5)HTU157:3.5)HTU154:9.5)HTU123:4.0)HTU109:4.0)HTU124:6.0)HTU102:6.5)HTU151:5.5)HTU121:4.5)HTU150:6.5)HTU108:13.0)HTU153:2.0)HTU122:3.0,(Litharca_lithodomus:20.0,((Chama_sp_PR41:37.0,Lissarca_picta:22.0)HTU140:5.0,(Lissarca_rubrofusca_USNM17:21.5,((Limopsis_loringi_AU11:17.0,(Limopsis_forteradiata_AU01:12.5,(Aupouria_parvula:29.0,Limopsis_marionensis:12.0)HTU137:6.5)HTU115:4.5)HTU136:5.0,(Limopsis_woodwardi_AU26:11.0,(Limopsis_tenisoni_AU29:15.5,(Lissarca_notarcadensis_USNM14:18.5,((Lima_lima:28.5,((Bentharca_asperula_USNM68:25.5,Isognomon_radiatus_PR29:32.0)HTU131:11.5,(Lima_caribaea:29.0,(Isognomon_alatus_PR44:25.0,(Limopsis_aurita_BMNH07:13.0,(Bentharca_nodulosa:19.5,(Arcopsis_adamsi:26.5,Limopsis_diegensis:15.5)HTU135:5.5)HTU114:4.5)HTU134:11.0)HTU104:5.0)HTU132:2.0)HTU113:2.5)HTU133:9.5,(Nipponarca_bistrigata:24.5,(((Lithophaga_plumula:44.5,Malleus_candeanus_PR31:29.5)HTU143:12.5,(Parallelodontinae:33.5,(Noetia_ponderosa_FLA01:23.5,(Anadara_geissei_CV01:15.5,Philobrya_setosa_USNM41:27.0)HTU147:9.5)HTU119:5.5)HTU146:5.5)HTU106:5.5,(Cosa_parallelogramma_USNM26:21.0,Melaxinaea_labyrintha:26.5)HTU144:7.5)HTU118:4.5)HTU145:7.0)HTU101:5.0)HTU139:8.0)HTU116:2.0)HTU138:2.5)HTU105:10.0)HTU141:3.5)HTU117:10.0)HTU142:11.0)HTU130:3.0)HTU128:5.0)HTU111:4.5)HTU127:4.5)HTU103:4.0)HTU129:3.0)HTU112:17.0,(Adacnarca_nitens:17.0,((Anadara_transversa:21.5,(((Fugleria_illota:13.0,(((Glycymeris_pilosa_FMNH06:20.0,(Glycymeris_glycymeris_MNHN120:18.0,Tucetona_pectunculus:22.0)HTU162:3.5)HTU163:11.0,(Anadara_ovalis:15.5,(Hyotissa_hyotis_CR48:29.5,((Brachidontes_domingensis_PR26:28.5,Lithophaga_aristata:15.0)HTU178:4.0,((Limaria_pellucida:46.5,((Isognomon_bicolor_PR33:27.5,Pteria_colymbus_PR37:15.5)HTU181:6.0,(Pinctada_mazatlantica_CR49:18.5,Pteria_hirundo:27.0)HTU183:4.0)HTU182:3.5)HTU180:7.5,(Atrina_rigida:12.5,Pinna_carnea_PR24:13.0)HTU185:7.5)HTU184:9.0)HTU177:8.5)HTU176:14.5)HTU175:5.5)HTU164:8.5,(Scaphula_celox_:24.0,Scaphula_deltae_BMNH16:21.0)HTU166:9.0)HTU165:6.5)HTU167:6.5,(Barbatia_domingensis:14.0,(Barbatia_amygdalumtostum:11.5,(Barbatia_candida_PR04:13.5,Bentharca_xenophoricola_FMNH13:23.0)HTU170:7.5)HTU169:7.0)HTU174:7.5)HTU168:6.5,(Striarca_interplicata_USNM49:20.0,(Arcopsis_afra:11.5,Arcopsis_solida:9.5)HTU173:6.5)HTU172:10.0)HTU171:5.0)HTU179:14.5,(Mytilid_sp:47.0,(Cosa_waikikia:22.0,(Cosa_costata:20.5,Cosa_scabra_AU155:17.0)HTU189:3.5)HTU188:4.5)HTU187:5.0)HTU186:7.0)HTU190:8.0)HTU198:4.5)HTU161:6.5)HTU192:8.0)HTU193:7.0)HTU194:6.0)HTU195:7.0)HTU196:3.0)HTU197:8.0)HTU100[1164.0];
